.docker/entrypoint/lib/commands/jekyll_commander.rb

```diff
@@ -9,29 +9,37 @@
 module Jekyll
   # The Jekyll::PlantUml module contains everything related to Jekyll::PlantUml.
   module PlantUml
-    # The Jekyll::PlantUml::JekyllCommander class executes Jekyll commands such
-    # as `build` and `serve` against the correct `Jekyll::Commands::*` class.
-    class JekyllCommander
-      def initialize(jekyll_config)
-        @jekyll_config = jekyll_config
-      end
+    # The Jekyll::PlantUml::Commands module contains the implementations of the
+    # various commands that Jekyll PlantUML supports.
+    module Commands
+      # The Jekyll::PlantUml::JekyllCommander class executes Jekyll commands such
+      # as `build` and `serve` against the correct `Jekyll::Commands::*` class.
+      class JekyllCommander
+        def initialize(jekyll_config)
+          raise ArgumentError, 'jekyll_config cannot be nil' if jekyll_config.nil?
```

Comment:
Codacy found an issue: [Line is too long. [82/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5768825)